tests/integ/test_huggingface_torch_distributed.py

```diff
@@ -0,0 +1,53 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+from __future__ import absolute_import
+
+import os
+import pytest
+from sagemaker.huggingface import HuggingFace
+from tests.integ import timeout
+from tests.integ import DATA_DIR, TRAINING_DEFAULT_TIMEOUT_MINUTES
+
+
+@pytest.mark.release
```

Comment:
Could we remove the mark as release? This makes test run in all regions, and I worry that instance type may not be available in all regions. Test will still run in us-west-2.

Reply:
Sorry I didn't catch this earlier, once this is removed I can make sure it's merged this week